bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
If you're curious, I figured out what was causing the ValueError. The order of my decorators was the problem. I assumed the order in which you placed the patch was the order in which you added the parameter.
E.g.)

````
@mock.patch('some.thing.a')
@mock.patch('some.thing.b')
@mock.patch('some.thing.c')
def test_alphabet(a, b, c)
```
But it's really 
````
@mock.patch('some.thing.c')
@mock.patch('some.thing.b')
@mock.patch('some.thing.a')
def test_alphabet(a, b, c)
```

Well, now I know. :)